daml/Common.daml

```diff
@@ -71,10 +71,10 @@ onboardProviders = do
   operatorRoleCid           <- submit operator  do createCmd Operator.Role with observers = fromList [public]; ..
 
   custodianRoleOfferCid     <- submit operator  do exerciseCmd operatorRoleCid Operator.OfferCustodianRole with provider = bank
-  custodianRoleCid          <- submit bank      do exerciseCmd custodianRoleOfferCid Custodian.Accept
+  (custodianRoleCid, issuanceServiceCid, custodyServiceCid)          <- submit bank      do exerciseCmd custodianRoleOfferCid Custodian.Accept
```

Comment:
I would get rid of the whitespaces here since it's no being used to align with the above line...you could also extend the whitespace on line 73 to match this one if it looks nice.